test/Microsoft.AspNetCore.JsonPatch.Test/ObjectAdapterTests.cs

```diff
@@ -756,19 +756,35 @@ public void DeserializationMustWorkWithoutEnvelope()
         }
 
         [Fact]
-        public void DeserializationMustFailWithEnvelope()
+        public void DeserializationMustFailWithEnvelope_ForSimpleObject()
         {
             // Arrange
             var serialized = "{\"Operations\": [{ \"op\": \"replace\", \"path\": \"/title\", \"value\": \"New Title\"}]}";
 
             // Act & Assert
-            var exception = Assert.Throws<JsonPatchException>(() =>
+            var exception = Assert.Throws<JsonSerializationException>(() =>
             {
                 var deserialized
                     = JsonConvert.DeserializeObject<JsonPatchDocument<SimpleObject>>(serialized);
             });
 
-            Assert.Equal("The type 'JsonPatchDocument`1' was malformed and could not be parsed.", exception.Message);
+            Assert.Equal("The JsonPatchDocument was malformed and could not be parsed.", exception.Message);
+        }
+
+        [Fact]
+        public void DeserializationMustFailWithEnvelope()
```

Comment:
These test names are really silly. Can you come up with something more in line with our guidelines.